analytics/tests/test_counts.py

```diff
@@ -747,6 +749,40 @@ def test_active_users_log_by_is_bot(self) -> None:
         self.assertEqual(1, RealmCount.objects.filter(property=property, subgroup=False)
                          .aggregate(Sum('value'))['value__sum'])
 
+    def test_invites_sent(self) -> None:
+        property = 'invites_sent::day'
+
+        def assertInviteCountEquals(count: int) -> None:
+            self.assertEqual(count, RealmCount.objects.filter(property=property, subgroup=None)
+                             .aggregate(Sum('value'))['value__sum'])
+
+        user = self.create_user(email='first@domain.tld')
+        stream, _ = self.create_stream_with_recipient()
+        do_invite_users(user, ['user1@domain.tld', 'user2@domain.tld'], [stream])
+        assertInviteCountEquals(2)
+        # We currently send emails when re-inviting users that haven't
+        # turned into accounts, so count them towards the total
+        do_invite_users(user, ['user1@domain.tld', 'user2@domain.tld'], [stream])
+        assertInviteCountEquals(4)
+        # Test mix of good and malformed invite emails
```

Comment:
Small style bit: I think this test would be easier to read with a blank line between each `assert...` and the following comment. Basically it has the structure of a bunch of cases with explanation, action, and check, and it'd be easier to perceive and scan through that structure with a bit of visual separation between each case and the last.